pvr.iptvsimple/resources/language/resource.language.en_gb/strings.po

```diff
@@ -350,3 +408,52 @@ msgstr ""
 msgctxt "#30685"
 msgid "Note this can only apply to http/https streams that are processed by libavformat (e.g. M3u8/HLS). Using libavformat can be specified in an M3U file by setting the property `inputstreamclass` as `inputstream.ffmpeg`. I.e. adding the line: `#KODIPROP:inputstreamclass=inputstream.ffmpeg`. If this opton is not enabled it can still be enabled per stream/channel by adding a kodi property, i.e.: `#KODIPROP:http-reconnect=true`."
```

Comment:
> Using libavformat can be specified in an M3U file by setting the property `inputstreamclass` as `inputstream.ffmpeg`. I.e. adding the line: `#KODIPROP:inputstreamclass=inputstream.ffmpeg`. 

inputstream.ffmpegdirect ?

> If this opton is not 

typo. option

Reply:
Either actually. One is kodi's internal impl the other the addon.

Reply:
I've updated the text to reflect that.

Reply:
👍